test/protocol/modules/navIssuanceModule.spec.ts

```diff
@@ -3323,6 +3326,140 @@ describe("NavIssuanceModule", () => {
       });
     }
   });
+
+  context("custom set valuer", () => {
```

Comment:
Let's move the respective issue and redeem tests into the `#issue` and `#redeem` blocks. Also add a test in the `#initialize` function that checks that a custom SetValuer is properly set if it's passed in